server/routes/notification.js

```diff
@@ -107,9 +112,9 @@ module.exports = (logger, DEPRECATEnotificationService) => (router) => {
         const offset = (page - 1) * perPage
 
         Promise.all([
-          DEPRECATEnotificationService.getShiftNotificationsCount(req.user.username),
```

Comment:
Whats happened with the renaming back?

Reply:
The service has been added to expresses `app`. It is added here under  the file `DEPRECATEnotificationService` is added under the name `notificationService`. This should allow us to build this so that we can toggle between the two services. The routes should not care which service we give, so I have removed the previously deprecated naming from them.

See my other comment for where this will be added

Reply:
👍